portfolio/src/main/webapp/script.js

```diff
@@ -86,9 +86,8 @@ function makeComment(heading, content) {
 function getDropdownVal() {
     const dropdown = document.getElementById("num-comments");
     dropdown.onchange = changeDropdownVal;
-    if (sessionStorage["num-comments"]) {
-        dropdown.value = sessionStorage["num-comments"];
-    }   
+    sessionStorage["num-comments"] = sessionStorage["num-comments"] || 10;
```

Comment:
This is a good use of `a = b||c`, but in general I you don't want magic numbers floating around the code like this. I would define all the magic numbers like this in a single place and reference them from there. 0's and 1's are usually ok, but other numbers not so much.

Reply:
Ok, i changed them to variables!